core/templates/dev/head/i18n.js

```diff
@@ -45,20 +45,26 @@ oppia.constant('DEFAULT_TRANSLATIONS', {
 });
 
 oppia.controller('I18nFooter', [
-    '$rootScope', '$scope', '$translate',
-    function($rootScope, $scope, $translate) {
+    '$http', '$rootScope', '$scope', '$translate',
+    function($http, $rootScope, $scope, $translate) {
   // Changes the language of the translations.
+  var _PREFERENCES_DATA_URL = '/preferenceshandler/data';
```

Comment:
Note: we're moving away from underscores before variable names in JS (instead, we're treating things declared as "var" as private, and things declared on "$scope" as being shared with the corresponding HTML template). So I suggest dropping the leading underscores.